### PR TITLE
feat: delete flow

### PR DESCRIPTION
**Pull request by 121watts**
Closes #influxdata/idpe#8674

Kapture 2020-09-23 at 13 36 07

### CODE REVIEW

**Comment by drdelambre** [comment]
LOOK AT THAT ROUND NUMBER

**Comment by 121watts** [comment]
> LOOK AT THAT ROUND NUMBER

💯

File: package.json [hunk 1]
```diff
@@ -107,6 +109,7 @@
     "html-webpack-plugin": "^3.2.0",
     "http-proxy-middleware": "^0.18.0",
     "identity-obj-proxy": "^3.0.0",
+    "intermock": "^0.2.2",
```
Comment (drdelambre) [inline package.json:112]: we're not importing this anywhere. needed?

File: jestSetup.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ jest.mock('honeybadger-js', () => () => null)
 
 // cleans up state between @testing-library/react tests
 afterEach(() => {
+  window.localStorage.clear()
```
Comment (drdelambre) [inline jestSetup.ts:29]: noice

**Review by drdelambre — APPROVED** [review]

**Pull request merged**